databuilder/databuilder/extractor/redshift_metadata_extractor.py

```diff
@@ -29,7 +29,7 @@ def get_sql_statement(self, use_catalog_as_cluster_name: bool, where_clause_suff
             cluster_source = f"'{self._cluster}'"
 
         if where_clause_suffix:
-            if where_clause_suffix.lower().startswith("where"):
+            if "where" in where_clause_suffix.lower():
```

Comment:
is this change in case there is leading whitespace? I'd recommend instead using `lstrip()` if so, and then you can still check that the string starts with `where`